src/physics_mg.cpp

```diff
@@ -64,7 +64,12 @@ void sample_reaction(Particle& p)
 
   // Play Russian roulette if survival biasing is turned on
   if (settings::survival_biasing) {
-    if (p.wgt() < settings::weight_cutoff) {
+    double weight_cutoff = settings::weight_cutoff;
+    // if survival normalization is applicable, use normalized weight cutoff
+    if((settings::source_file || settings::surf_source_read)&&(settings::survival_normalization)){
+      weight_cutoff  = p.wgt_cutoff();
+    }
+    if (p.wgt() < weight_cutoff) {
```

Comment:
```suggestion
    if (p.wgt() < weight_cutoff*p.wgt0()) {
```

Reply:
I think adding an extra multiply by 1 procedure here in the base case of unbiased sources is cleaner than adding additional settings.